src/arm/overlays/BBAI-4D4C-00A1.dts

```diff
@@ -15,7 +15,7 @@
  
 &{/chosen} {
 	overlays {
-		BBAI-BB-BONE-4D4C-01-00A1 = __TIMESTAMP__;
+		BBAI-4D4C-00A1 = __TIMESTAMP__;
```

Comment:
Shouldn't this be non-BeagleBone-AI specific?

Reply:
Sorry this one only works for BBAI, but we should merge this. Even when I create a compatible overlay this overlay will be the go to resource for new LCD cape overlays and will help to understand what's going on inside compatible LCD overlays because things from this will go to several files when compatibility layer will support LCDs.

Reply:
Agree we should merge, but please document the current limitations so it can be picked up in the future.